### PR TITLE
Issue #405: GreenSocket.accept does not notify_open

### PR DESCRIPTION
**Pull request by orishoshan**

### CODE REVIEW

**Comment by temoto** [comment]
Thank you. Great job with tests.

Please also fix PEP8 issues.
Please don't use `str %` interpolation, `str.format()` instead.

```
tests/greenio_test.py:337:13: E303 too many blank lines (2)
            server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
            ^
tests/greenio_test.py:348:102: E501 line too long (118 > 101 characters)
                                                             "notified %d times" % patched_mark_as_reopened.call_count
                                                                                                     ^
tests/greenio_test.py:366:13: E303 too many blank lines (2)
            server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
            ^
tests/greenio_test.py:379:102: E501 line too long (118 > 101 characters)
                                                             "notified %d times" % patched_mark_as_reopened.call_count
                                                                                                     ^
```

File: tests/greenio_test.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import socket as _orig_sock
 import sys
 import tempfile
+import mock
```
Comment (temoto) [inline tests/greenio_test.py]: mock is bundled in tests directory.
`import tests.mock`

**Comment by orishoshan** [comment]
Fixed PR comments.

**Comment by temoto** [comment]
You're not watching Travis build, right? :-)

**Comment by orishoshan** [comment]
Whoops. I was sure I'd fixed that one... and no, obviously :)

**Comment by temoto** [comment]
Thank you.

**Pull request merged**